exporter/signalfxexporter/config_test.go

```diff
@@ -58,17 +58,28 @@ func TestLoadConfig(t *testing.T) {
 	assert.Equal(t, defaultCfg, e0)
 
 	e1 := cfg.Exporters[config.NewComponentIDWithName(typeStr, "allsettings")]
+	maxConnections := 70
+	maxIdleConns := 300
+	maxIdleConnsPerHost := 500
+	idleConnTimeout := 15 * time.Second
 	expectedCfg := Config{
 		ExporterSettings: config.NewExporterSettings(config.NewComponentIDWithName(typeStr, "allsettings")),
 		AccessToken:      "testToken",
 		Realm:            "us1",
-		MaxConnections:   70,
+		MaxConnections:   &maxConnections,
 		Headers: map[string]string{
 			"added-entry": "added value",
 			"dot.test":    "test",
 		},
```

Comment:
Since we're adding a new one in the client settings this field should be deleted/migrated everywhere.

Reply:
Pushed the changes accordingly.